pkg/templates/loader/testdata/full_schema/schema.yaml

```diff
@@ -0,0 +1,76 @@
+openapi: 3.0.3
+info:
+  title: dev
+  version: v1.0.0
+components:
+  schemas:
+    variables:
```

Comment:
<img width="354" alt="image" src="https://github.com/seal-io/walrus/assets/5697937/8db9be49-7825-487b-ba7e-6da7efe19b42">

CRD schema is abbreviated to description and properties. Do you think we can do in similar way? Does it affect UI rendering?

Reply:
after communicating, keep the openAPI format and metadata